src/main/java/com/puppycrawl/tools/checkstyle/XdocsPropertyType.java

```diff
@@ -29,7 +29,6 @@
  * for clarification of the user-friendly type.
  * For example, some string array property may represent a set of tokens.
  *
```

Comment:
Please remove empty ending line for javadoc.

Reply:
Done